javascript-typing-tutor/main.js

```diff
@@ -0,0 +1,14 @@
+var $span = document.querySelectorAll('span');
+var spanElements = 0;
+document.addEventListener('keydown', myFunction);
+function myFunction(event) {
+  if (event.key !== $span[spanElements].textContent) {
+    $span[spanElements].className = 'red-color';
+    return;
+  }
+  if (event.key === $span[spanElements].textContent) {
```

Comment:
This condition is the opposite of the condition in the above conditional, so you can just make this an `else` block for that conditional.